### DISCUSSION TITLE
用户反馈V23系统占用过高

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4016] [Troubleshooting | 故障排查]
https://bbs.deepin.org/phone/zh/post/254749
【系统环境】
镜像版本：V23

内核版本：6.1.11-amd64-desktop-hwe
处理器 : Intel(R) N100 (四核 / 四逻辑处理器)
主板 : TB100
内存 : 2GB(Row of chips Synchronous 6400 MHz (0.2 ns) 6400 MT/s)/2GB(Row of chips Synchronous 6400 MHz (0.2 ns) 6400 MT/s)/2GB(Row of chips Synchronous 6400 MHz (0.2 ns) 6400 MT/s)/2GB(Row of chips Synchronous 6400 MHz (0.2 ns) 6400 MT/s)
显示适配器 : Intel Corporation
音频适配器 : Intel Corporation
存储设备 : WDC WDS240G2G0B-00EPW0 (240 GB)
蓝牙 : mini-PC
网络适配器 : RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller/Wireless 7265/RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller
鼠标 : Mi Wireless Combo (Xiaomi Mi Wireless Combo)
键盘 : Mi Wireless Combo (Xiaomi Mi Wireless Combo)
显示设备 : C270(27.0 英寸 (597x336 mm))
其他设备 : RTS5129 Card Reader Controller

【操作步骤】
deepin-kwi_x11 CPU占用高

**Comment by BLumia** [comment]
相关： https://github.com/linuxdeepin/deepin-kwin/pull/160
beta 应该会涵盖这个 PR
